### PR TITLE
Global replace to add \n and {} to 1 line if/else statements

### PR DESCRIPTION
**Pull request by bhoffman0**
I noticed some 1 line if statements like ``if (t > maxTemp) maxTemp = t;`` and did a global replace to add in newlines, spacing, and curly braces.  Proud of my regexp: s/([ ]*)((else if|if|else)\s*\(.*\))([^\n]*;)/$1$2\n$1{\n$1    $4\n$1}/g

### CODE REVIEW

File: _sources/TimedTests/test3.rst [hunk 1]
```diff
@@ -510,18 +510,30 @@ Click the |start| button when you are ready to begin the exam, but only then as
...
-             else if (num1>num2){result = 1;}
+             if (num1==num2)
+             {
+                  result = 0;
+             }
+             else if (num1>num2)
+             {
+                 result = 1;
+             }
              else {result = -1;}
```
Comment (gigamonkey) [inline _sources/TimedTests/test3.rst]: Should probably fix this too, right?
```suggestion
             else
             {
                 result = -1;
             }
```

File: _sources/TimedTests/test3.rst [hunk 2]
```diff
@@ -510,18 +510,30 @@ Click the |start| button when you are ready to begin the exam, but only then as
...
+             {
+                  result = 0;
+             }
              else {
-                if(num1 > num2) {result = 1;}
+                if(num1 > num2)
+                {
+                     result = 1;
+                }
                 else {result = -1;}
```
Comment (gigamonkey) [inline _sources/TimedTests/test3.rst]:
```suggestion
                else
                {
                    result = -1;
                }
```

**Review by gigamonkey — APPROVED** [review]
I fixed a few else's that maybe escaped your regexp because they had braces on one line.

File: _sources/TimedTests/test3.rst (hunk 1 above)
Comment (bhoffman0) [reply _sources/TimedTests/test3.rst]: Thanks!

**Pull request merged**